psalm.xml

```diff
@@ -73,6 +73,7 @@
             <errorLevel type="suppress">
                 <file name="src/Storage/Doctrine/EventListener/UpdateStoredMessageListener.php"/>
                 <file name="src/Storage/Doctrine/StoredMessage.php"/>
+                <file name="src/Storage/Doctrine/EventListener/SaveRetriedMessageListener.php"/>
```

Comment:
Suppressing entire files instead of individual methods by adding annotations inside the class itself? I've been talking about the pros and cons with someone else recently - what is your thinking?

Reply:
actually it's the first time i use psalm, so i'm not aware about all the way we can disable some checks.
Indeed, i feel like this is lacking some granularity this way. On the other hand, i don't really like polluting the code with vendor's annotations...
But my choice would go to annotations, otherwise, this is to much global => i'll open an issue about that